scripts/compile_coq.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/sh -e
+
+help() {
+  echo "./compile-coq <compiler path> <path to directory> [--opam opam] [-c|--compile-opam] [-r|--remove]"
```

Comment:
I'd use underscores instead of spaces in argument names to avoid any confusion, e.g.: `compiler_path`, and `path_to_directory`.

Reply:
Done